### DISCUSSION TITLE
How to setup prices in played game status message

### DISCUSSION THREAD

**Discussion by ghost** [Refloow/Steam-Card-Bot-PRO#46] [General]
Can someone show me how to set it up? I don't know what format to use to set, I can't understand the meaning of XX:X.![153479050-827bde79-f895-489b-bbba-ba8d9a9a6f13.png](https://user-images.githubusercontent.com/99354599/153627125-d8998dd1-93c9-49ee-b6fe-bfcc791600c2.png)

**Comment by Refloow** [comment]
These are prices you are displaying in custom played game status
XX:X CS:GO for example means number of sets for cs go key
If you are selling 15 sets for csgo key you type 15:1 CSGO

**Reply by ghost** [reply]
Thank you so much!
